Day-I-Social-Card/social-card/src/components/CardComponents/CardBanner.js

```diff
@@ -1,2 +1,12 @@
 import React from 'react';
 import './Card.css';
+
+//                                    CardBanner -> Display's this image as background -> https://ibin.co/3wnC6SgIOJud.png
+
+const CardBanner = (props) => {
```

Comment:
If you aren't using props, no need to pass them in 👍

Reply:
true